Sources/MongoKitten/Commands/FindCommand.swift

```diff
@@ -27,17 +27,17 @@ public struct CursorSettings: Encodable {
     var batchSize: Int?
 }
 
+struct CursorDetails: Codable {
```

Comment:
`GetMoreReply.CursorDetails` still exists, and it's basically the same struct as this `CursorDetails`. As you moved the definition outside of the `CursorReply` struct, you might also want to use this definition of the `CursorDetails` struct in `GetMoreReply`

Reply:
Oops! That was not intended. It was meant to be moved, not duplicated.

Reply:
Ah – they do differ. One of the `CursorDetails` has a variable named `firstBatch`, the other one named `nextBatch`. I'll move this `CursorDetails` back to where it was before.s